plugins/ci/pkg/ci/ci.go

```diff
@@ -402,7 +401,9 @@ func getConfigurationForClonedRepo() (string, string, *models.Configuration, err
 		return "", "", nil, errors.New("IMAGE_VERSION environment variable not set")
 	}
 
-	basePath := filepath.Join("/app", "repository")
+	// marker
+	basePath := filepath.Join("/var/tmp", "repository")
+	// basePath := filepath.Join("/app", "repository")
```

Comment:
I wonder if you left it commented to revert before merge?

Reply:
Ooooo yes, I did do that. Thank you!